packages/terra-button/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 ## Unreleased
+* Added
+  * Added new subcomponent `terra-icon-button` to display icon.
+
+* Breaking Changes
+  * `icon`, `isIconOnly` and `isReveresed` props have been moved from `terra-button` to `terra-icon-button`.
```

Comment:
`terra-icon-button` sounds like a package name. Should we be referencing it as `IconButton` instead?

Reply:
[updated](https://github.com/cerner/terra-core/pull/3640/commits/c5a73564b94b97cb0c54c2fcd01371579564e7c7#diff-b52d144e24775550035ddae340a31241b9f3b7f26c509d417ddb46e28f698ea4)